package.json

```diff
@@ -31,14 +30,12 @@
     "recoil": "^0.7.1",
     "regenerator-runtime": "^0.13.9",
     "sass": "^1.50.0",
-    "uuidv4": "^6.2.13",
```

Comment:
why we are removing this uuidv4?

Reply:
If I'm not mistaken we don't use it. We include v4 from lib uuid, not lib uuidv4.

Reply:
uuidv4 itself contains an inbuilt function for generating uuidv5 too and we r still using uuidv4 in creating goals, contacts, and all.